### ISSUE TITLE
Event: error in triggerEvent.call

### ISSUE THREAD

**Issue by laukstein** [sampotts/plyr#1623]
Noticed **Event: error** in `triggerEvent.call` on Chrome 65, Linux.
https://github.com/sampotts/plyr/blob/74e3990604698b1f2af3ed51e65b0ca820c1d624/src/js/listeners.js#L499
![grabilla em8812](https://user-images.githubusercontent.com/183966/69709685-89f17480-1106-11ea-961b-19a7666c318d.png)

**Comment by hasan-wajahat** [comment]
Yes Plyr library throws multiple unhandled errors which can be seen on error reporting apps. I have also opened an issue for this as the Plyr floods our error reporting channel.

**Comment by laukstein** [comment]
@hasan-wajahat, what is the issue number you open for it?

**Comment by hasan-wajahat** [comment]
@laukstein The issues I opened was #1581

**Comment by laukstein** [comment]
![grabilla em3900](https://user-images.githubusercontent.com/183966/91472332-f240d280-e89f-11ea-9e41-0fe008a862f0.png)

**Comment by sacki5** [comment]
I still have the same problem. Version 3.6.2
<img width="748" alt="Skärmavbild 2020-09-02 kl  16 15 16" src="https://user-images.githubusercontent.com/25466424/91994968-847d2680-ed37-11ea-85c0-d23f21162e8f.png">

**Comment by laukstein** [comment]
Still same issue on v3.6.3
